network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -447,7 +448,8 @@ def redundant_policy_text(self, policy):
 
     def exec(self):  # noqa: C901
         if not self.config:
-            return QueryAnswer(False, f'No NetworkPolicies in {self.config.name}. Nothing to check sanity on.', '', 1)
+            return QueryAnswer(False, f'No NetworkPolicies in {self.config.name}. Nothing to check sanity on.', '',
+                               query_not_executed=True)
```

Comment:
Hi @adisos,
when a config has no policies in sanity check. the current query output is 
> No NetworkPolicies in {self.config.name}. Nothing to check sanity on.', '',
so , I turned on the `query_not_executed`.
If it shouldn't be turned on, wouldn't be better to change the query output?